exonum/src/runtime/mod.rs

```diff
@@ -168,15 +255,19 @@ impl Caller {
     }
 }
 
+/// Provides the current state of the blockchain and caller information for the transaction
```

Comment:
I think either 'transaction being executed' or 'transaction which is being executed', but @Maria-Nosyk could say for sure.